tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/longlivedtoken/LongLivedTokenInjectKeys.java

```diff
@@ -0,0 +1,5 @@
+package com.linagora.tmail.james.jmap.longlivedtoken;
+
+public interface LongLivedTokenInjectKeys {
```

Comment:
Java in scala repo code? Make it a trait, and you could probably just move it to `JwtTokenRoutes` as well

Reply:
I have trouble with Scala, How can I create a final static String? 
When I using `@Named("JwtTokenRoutes.INJECTION_KEY")`, I got an exception: you should use a static string

Reply:
Please follow this pattern : `server/protocols/jmap-rfc-8621/src/main/java/org/apache/james/jmap/http/rfc8621/InjectionKeys.java`

 -> have the java place holders in a separate Java class, java folder
 -> the rest of the code can stay in scala

Reply:
I moved `LongLivedTokenInjectKeys` to `tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap` 
I don't sure it is right or not